tests/post_training/test_templates/test_smooth_quantize.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import abstractmethod
+from typing import Callable, Dict, TypeVar
+
+import pytest
+
+from nncf.parameters import ModelType
+from nncf.quantization.advanced_parameters import AdvancedQuantizationParameters
+from nncf.quantization.advanced_parameters import OverflowFix
+from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantization
+from tests.post_training.test_templates.helpers import LinearModel
+from tests.post_training.test_templates.helpers import get_static_dataset
```

Comment:
Could you please add a small test that will check that created tensor collectors have right reducer/aggregator classes and params?

Reply:
Added, thanks!